daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-overview.md

```diff
@@ -82,13 +85,94 @@ Logs for application and `daprd` are captured in separate files. These log files
 
 Even if you've decided to rename your resources folder to something other than `.dapr`, the log files are written only to the `.dapr/logs` folder (created in the application directory).
 
-
 ## Watch the demo
 
 Watch [this video for an overview on Multi-App Run](https://youtu.be/s1p9MNl4VGo?t=2456):
 
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/s1p9MNl4VGo?start=2456" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
+{{% /codetab %}}
+
+{{% codetab %}}
+<!--kubernetes-->
+
+## Multi-App Run template file
+
+Generate the multi-app template file (`dapr.yaml`) by running one of the following commands:
```

Comment:
This will not generate the multi-app run template file ... 
Based on the already existing `dapr.yaml` multi-app run template file, the the commands `dapr run -k -f .` or `dapr run -k -f dapr.yaml` will start the applications defined in the template in k8s `default` namespace.